examples/Foo.py

```diff
@@ -0,0 +1,16 @@
+from aperturedb.transformers.transformer import Transformer
+
+
+class Foo(Transformer):
```

Comment:
Seriously! Foo bar! we compiler ppl now? :)

Reply:
This is just to give an example in docs as to what this is.

Reply:
I've updated some docstrings. will add more description once this is merged.